main.js

```diff
@@ -2,6 +2,12 @@ const hambuMenu = document.querySelector('.hambu-menu');
 const closeMenu = document.querySelector('.x-img');
 const menuToggle = document.querySelector('#menu-toggle');
 const hambuChilds = document.querySelectorAll('.hambu-child');
+const seeproject1 = document.querySelector('#buttonds1');
+const seeproject2 = document.querySelector('#buttonds2');
+const seeproject3 = document.querySelector('#buttonds3');
+const seeproject4 = document.querySelector('#buttonds4');
+const popupWindow = document.querySelector('#popup');
+// const closewindow = document.querySelector('.close');
 
 function show() {
```

Comment:
- [ ] <b>REQUIRED:</b> Keep your code clean by making sure to remove all commented-out code.
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

Reply:
> * [x]  REQUIRED: Keep your code clean by making sure to remove all commented-out code.
> * [ ]  REQUIRED: Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
> 
> ```js
>   const functionName = () => {
>      // code here
>   }
> ```

- [ ] As the CR mentioned, ES6 arrow function is preferred over the function keyword. It keeps your code clean and easy to read. Kindly work on this